src/components/about/AboutComponent.tsx

```diff
@@ -0,0 +1,23 @@
+import aboutImg from '@/assets/about.png';
+
+import AboutCSS from './about.module.css';
+
+export const AboutComponent = () => (
+    <div className={AboutCSS.container}>
+        <img className={AboutCSS.img} src={aboutImg} alt="AboutIMG" />
+        <div className={AboutCSS.text}>
+            <h1 className={AboutCSS.title}>About me</h1>
+            <p className={AboutCSS.description}>
+                Hi! My name is Oleh, and I&apos;m a Junior Frontend Developer. I am already familiar with main Web Technologies like React,
+                HTML, CSS, JavaScript and Git version control system. <br /> <br />
```

Comment:
You should avoid this type of indentation. You need to create three `<p>` blocks, which will be having indentation as in the design.